### PR TITLE
Added last row margin features

### PR DESCRIPTION
**Pull request by KevinTaron**
#177 I've added a checkbox in settings so a user can activate "margin in last row" for every site.
If the user only want a margin in last row he can overwrite it with the row styles like @jordif did.

(Need that feature on some of my sites. At the moment i'm adding a empty row on the bottom to each page for this feature)

### CODE REVIEW

File: siteorigin-panels.php [hunk 1]
```diff
@@ -719,12 +720,19 @@ function siteorigin_panels_generate_css($post_id, $panels_data = false){
 			}
 		}
 
-		// Add the bottom margin to any grids that aren't the last
-		if($gi != count($panels_data['grids'])-1){
+		if($panels_margin_bottom_last_row) {
```
Comment (braamgenis) [inline siteorigin-panels.php]: You could accomplish the same thing by just adding ` || !empty( $panels_margin_bottom_last_row )` to the original conditional expression. So in future, if anything needs to change about how we're calling `$css->add_row_css` we only have to change it in one place.

**Comment by braamgenis** [comment]
Hi @KevinTaron Thanks for your contribution! I just have one suggestion about your code. Thanks again! :+1:

**Comment by KevinTaron** [comment]
Hi @braamgenis you are totaly right. Don't know why did that this way ... Fixed ;)

**Comment by braamgenis** [comment]
Great, merging!

**Pull request merged**